### PR TITLE
AppUI: Prompt to restart the app when changing the game path 

### PR DESCRIPTION
**Pull request by barkermn01**
Added the same code that has just been added to 7th heaven so when a user changes a game path it recommend and asks the user to restart junction 8

### CODE REVIEW

File: AppCore/StringKey.cs
Comment (julianxhokaxhiu) [inline AppCore/StringKey.cs:1]: Can you please find a way to highlight only your change in this file?

Comment (barkermn01) [reply AppCore/StringKey.cs:1]: i hate git & windows i explicitly told it to not convert line endings...

File: AppCore/StringKey.cs [hunk 1]
```diff
@@ -1,653 +1,654 @@
...
+        ControlNameContainsInvalidChars,
+        ControlsWithThatNameAlreadyExist,
+        SuccessfullyCreatedCustomControls,
+        FailedToCreateCustomControls,
+        SuccessfullyDeletedCustomControls,
+        FailedToDeleteCustomControls,
+        EnterNameForCatalog,
+        GeneralSettingsHaveBeenUpdated,
+        SettingsNotValid,
+        Ff8ExePathChanged,
```
Comment (barkermn01) [inline AppCore/StringKey.cs:121]: This is the only change to this file i added this line

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thanks a lot!

**Pull request merged**